exercises/practice/intergalactic-transmission/.docs/introduction.md

```diff
@@ -0,0 +1,10 @@
+# Introduction
+
+Trillions upon trillions of messages are exchanged between Earth and the neighboring galaxies every millisecond. 
+But transmission over such long distances faces many obstacles.
+Pesky solar flares, worm holes, temporal distortions, stray forces, heck even the flap of a space butterfly wing can cause a random bit to flip during transmission.
+The difference between an "o" and a "k", a "[" and "{" or a cowboy emoji and a clown emoji can have huge ramifications, from missed orders to Galactic Stockmarket crashes.
+It is important for the receiver to know when the message has been corrupted so that the receiver can ask for the message to sent again.
```

Comment:
```suggestion
It is thus important for the receiver to know if a message has been corrupted so that the receiver can ask for the message to be sent again.
```